MAiD - Case Management App/.vscode/settings.json

```diff
@@ -3,6 +3,5 @@
     "**/node_modules": true,
     "**/bower_components": true,
     "**/.sfdx": true
-  },
-  "salesforcedx-vscode-core.show-cli-success-msg": false
```

Comment:
Get latest from Dev branch

Reply:
Changed settings.json just as it is in dev